github_pr.py

```diff
@@ -6,6 +6,17 @@
 from tabulate import tabulate
 from github import Github
 
+class NoApproversError(Exception):
+    def __init__(self, *args, **kwargs):
+        Exception.__init__(self, *args, **kwargs)
+        self.msg = args
```

Comment:
you don't need to define custom `__init__()`s. adding `pass` to the body is sufficient:
```python
class NoApproversError(Exception):
    pass
```